apps/client/src/components/Pages/recherche/SearchResults/SearchResults.tsx

```diff
@@ -152,6 +150,24 @@ const SearchResults = (props: Props) => {
             </div>
           </>
         )}
+        {filteredResults.suggestions.length > 0 && (
```

Comment:
I think you need to call the [getDisplayRuleForQuery](https://github.com/refugies-info/karfur/blob/af6d5d147ea35aef9864c4961357d1547fa75c6c/apps/client/src/lib/recherche/queryContents.ts#L39) function to know if the suggestions should be displayed or not.  For instance:

```
getDisplayRuleForQuery(query, "suggestions")
```

on top of checking if the suggestions item is field.

Reply:
For instance the suggestions shouldn't be shown in the case where there is a location and keywords:

<img width="1433" alt="image" src="https://github.com/user-attachments/assets/74f15433-21f8-4d74-92b8-f7317f92fc53">

Reply:
Thanks for the feedback. I just pushed the fix, is it what you had in mind?